msteams-platform/sbs-file-handling-in-bot.yml

```diff
@@ -0,0 +1,327 @@
+       You have successfully set up a bot in Azure Bot Service and now you need to set up bot service connection.
+
+- title: Set up bot service connection
+  durationInMinutes: 5
+  content: |
+    To ensure that the Bot Service enables user to sign in and access the bot, you must configure SSO for your bot. 
+    To set up bot service connection, perform the following:
+    
+    1. Add Redirect URI to the Azure bot resource
+    1. Expose API endpoint
+    1. Authorize client applications
+    1. Add any necessary API permissions for downstream calls
+    1. Enable implicit grant
+    1. Update manifest
+    1. Set up Bot Service connection
+
+    **To add Redirect URI to the Azure bot resource:**
+
+    1. Select **Manage** next to **Microsoft App ID**
+
+         ![Manage Bot](~/assets/images/teams-file-upload-bot/manage-sec.png)
+
+    1. Select **Authentication** from the left pane of your bot resource.
+    1. In **Platform configurations**, select **Add a platform**.
+    1. Select **Web**.
+    1. Enter **Redirect URIs** as **https://token.botframework.com/.auth/web/redirect** and select **Configure**.
+
+        ![Configure web redirect url](~/assets/images/teams-file-upload-bot/authentication.png)
+
+        The Redirect URI is added to the Azure bot resource.
+
+        ![Display web redirect url](~/assets/images/bots/redirectURIadded.png)
+        
+    **To expose API endpoint:**
+
+    1. Select **Expose an API** from the left pane.
+    1. Select **Set** for the Application ID URI.
+    1. Use the **Application ID URL** shown and append **api://** instead of **https://** in the **ngrok url** and add at the beginning of the App ID URL.
+
+        ![App ID URI](~/assets/images/teams-file-upload-bot/expose-api.png)
+
+    1. Select **Save**.
+    1. Select **Add a scope**.
+    1. In **Scope name**, enter **access_as_user**.
+    1. In **Who can consent?**, switch to **Admins and users**.
+    1. Enter the following values in the boxes:
+
+        | Field | Value |
+        | ----- | ----- |
+        | Admin consent display name | Teams can access the user’s profile |
+        | Admin consent description | Allows Teams to call the app’s web APIs as the current user. |
+        | User consent display name | Teams can access your user profile and make requests on your behalf |
+        | User consent description | Enable Teams to call this app’s APIs with the same rights that you have. |
+
+    1. Ensure that **State** is set to **Enabled**.
+
+        ![Add a scope](~/assets/images/teams-file-upload-bot/expo.png)
+
+    1. Select **Add scope**. The scope name automatically matches the Application ID URI, with **/access_as_user** added to the end.
+
+    **To authorize client applications:**
+
+    Add the following IDs as **Authorized client applications**, select the check box for **Authorized scopes** and select **Add application**.      
+    * 1fec8e78-bce4-4aaf-ab1b-5451cc387264 (Teams mobile or desktop application)            
```

Comment:
Are these the exact IDs to be used or will a user get different set of IDs. If this is representative, then obscure these or mention that this is an example and actual ID will be different.

@v-silakshmi

Reply:
made changes as advised.